csi-driver-templates/templates/pods/_quobyte_csi_node_driver_pod.tpl

```diff
@@ -34,6 +34,5 @@ spec:
       {{- if trim .Values.quobyte.podKiller.dnsPolicy }}
       dnsPolicy: {{ trim .Values.quobyte.podKiller.dnsPolicy  }}
       {{- end }}
-      {{- include "quobyte-csi-driver.nodeDriverPodVolumeAttachments" . | indent 6 }}
```

Comment:
This line is duplicated with the line https://github.com/quobyte/quobyte-csi-driver/pull/93/files#diff-d63dd174feb2a022ef6ad1db46e402d6aaebc86a58b764b4d02fd9ba19d497a8R33